dataland-frontend/src/components/pages/RequestData.vue

```diff
@@ -2,201 +2,202 @@
   <AuthenticationWrapper>
     <TheHeader />
```

Comment:
If a bulk request was not succesful, should there be a way to go back to the request page? Or will it be implemented in the dialog story?
![image](https://github.com/d-fine/Dataland/assets/104447806/9ee6aaf7-d9c0-4a77-8438-41dec86ea705)

Reply:
Yeah I wasn't sure about this either. Our old code had a button to the companies, but figma has it back to requests.
Easy enough to just change the button label and url.
What do you think?

Reply:
I would vote for changing it to take the user back to the request page

Reply:
@Worben @emanueldfine @PaulChristiansen has someone started this?

Reply:
I am now in charge of this problem.